math/mcmcf90/Portfile

```diff
@@ -0,0 +1,57 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           compiler_blacklist_versions 1.0
+PortGroup           compilers 1.0
+PortGroup           github 1.0
+PortGroup           makefile 1.0
+
+github.setup        mjlaine mcmcf90 95fb3b4eba55ac834a7285d83063e5e5eba263b2
+version             2019.01.17
+revision            0
+categories          math science
+maintainers         {@barracuda156 gmail.com:vital.had} openmaintainer
+license             MIT
+description         Fortran library for MCMC calculations
+long_description    This Fortran 90 library can be used to do Markov chain Monte Carlo simulation \
+                    from a posterior distribution of unknown model parameters defined \
+                    by a likelihood function and prior. The likelihood is given \
+                    as “sum-of-squares” difference of observed values from modelled values.
+homepage            https://mjlaine.github.io/mcmcf90
+checksums           rmd160  4135dd4b51e5afea3eecf3f4e5bf53c1a3d67d3a \
+                    sha256  420b4e064ea345c1b5d65a8c596febbe9f341dfd5a2abade36f5a17e179cee97 \
+                    size    56099
+
+compilers.choose    fc f90
+compilers.setup     require_fortran
+
+compiler.blacklist-append \
+                    *gcc-4.* {clang < 421}
+
+depends_lib-append  path:lib/libopenblas.dylib:OpenBLAS
+
+patchfiles          0001-Makefile-restore-PPC-support.patch
+
+post-patch {
+    reinplace "s|F90=gfortran|F90=${configure.fc}|g" ${worksrcpath}/mac.mk
+    reinplace "s|F77=gfortran|F77=${configure.fc}|g" ${worksrcpath}/mac.mk
+    reinplace "s|F90=gfortran|F90=${configure.fc}|g" ${worksrcpath}/maci.mk
+    reinplace "s|F77=gfortran|F77=${configure.fc}|g" ${worksrcpath}/maci.mk
+
+    platform darwin 10 powerpc {
+        # Rosetta hack:
+        reinplace "s|maci.mk|mac.mk|g" ${worksrcpath}/Makefile
+    }
+}
+
+use_parallel_build  no
+
+destroot {
+    copy ${worksrcpath}/libmcmcrun.a ${destroot}/${prefix}/lib/
```

Comment:
is there no way to build a dynamic library instead of a static one?

Reply:
This I can try implementing. Will update if it works.

Reply:
you want to try and do this or do you consider this PR finished?

Reply:
I will update on this soon. If it does not work, we can just merge. It just went forgotten.

Reply:
Done, I guess.